### PR TITLE
feat(convert): Added whitespace options

### PR DESCRIPTION
**Pull request by itpropro**
For a clean history, please merge #3 first.
This PR

- Added whitespace options with backwards compat
- Added whitespace tests
- Fixed a bug with a missing `:` at `green_heart`
- Added eslint TypeScript dev packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "convert-gitmoji",
-  "version": "0.1.0",
+  "version": "0.0.3",
```
Comment (Intevel) [inline package.json]: Same here, please don't change the version.

**Review by Intevel — CHANGES_REQUESTED** [review]
Can you please update the documentation in due to your changes in `src/index.ts`

**Comment by itpropro** [comment]
Updated the docs @Intevel

**Review by Intevel — APPROVED** [review]

**Comment by Intevel** [comment]
Thanks for your great pull requests @itpropro, released it! 💚

**Pull request merged**